Lib/pprint.py

```diff
@@ -233,6 +233,34 @@ def _pprint_ordered_dict(self, object, stream, indent, allowance, context, level
 
     _dispatch[_collections.OrderedDict.__repr__] = _pprint_ordered_dict
 
+    def _pprint_dict_view(self, object, stream, indent, allowance, context, level, items=False):
+        key = _safe_tuple if items else _safe_key
+        write = stream.write
+        write(object.__class__.__name__ + '([')
+        if self._indent_per_level > 1:
+            write((self._indent_per_level - 1) * ' ')
+        length = len(object)
+        if length:
+            if self._sort_dicts:
+                entries = sorted(object, key=key)
+            else:
+                entries = object
+            self._format_items(entries, stream, indent, allowance + 1,
+                                    context, level)
+        write('])')
+
+    _dict_keys_view = type({}.keys())
```

Comment:
There's a PR to add `DictKeysType` and friends: https://github.com/python/cpython/pull/30630/files#diff-14c8ad728bd63143566c2c1e5a85a7b20cebc910278ac9202978e845dc63f215R52-R54

It might be helpful here in the future.

Reply:
Thanks! Maybe it would be neater to rename the proposed private attrs as `_DictKeysType` and friends, just until  #30630 lands?

Reply:
IMO that is not necessary.